plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -66,7 +66,9 @@ flex.page.title=App Engine Deployment - Flexible Environment
 use.config.values=Use these values during deployment
 
 #Deploy properties page
-invalid.deploy.page.state=Only appropriate for projects with an {0} facet.
+add.facet.link=<a>Open the Project Facets page</a> and add the App Engine Java Standard \
+Environment or the App Engine Java Flexible Environment facet to enable the App Engine Deployment \
+settings.
```

Comment:
+1, since the current wording may give the impression that users have to take an immediate action, while they won't understand what it means most of the time. I also agree we don't really need to provide a shortcut link to the facet panel.